web/src/components/ThemeSettings/ThemeSettings.js

```diff
@@ -0,0 +1,76 @@
+import { Toaster } from '@redwoodjs/web/toast'
+const ThemeSettings = ({ data, update, loading }) => {
+  const handleClick = (e) => {
+    let themeSelection
+    if (e.target.value === 'theme2') {
+      themeSelection = '2'
+    } else {
+      themeSelection = '1'
+    }
+    update({
+      variables: {
+        id: data.restaurantByOwnerId.id,
+        input: {
+          theme: themeSelection,
+        },
+      },
+    })
+  }
+
+  return (
+    <>
+      <div className="flex">
+        <Toaster />
+        <div className="m-10 mr-20">
+          <div
+            className="px-64 py-40 border-green-700 border-4 shadow-2xl"
+            style={{
+              backgroundImage: 'url(/template2.png)',
+              backgroundPosition: 'center',
+              backgroundSize: 'cover',
+            }}
+          ></div>
+
+          <button
+            onClick={handleClick}
```

Comment:
We'll want to update this so that we pass the theme selection to the function.

```suggestion
            onClick={() => selectTheme('umamiMichelin')}
```